cli/version.go

```diff
@@ -1,3 +1,3 @@
 package cli
 //go:generate bash ./generate_version.sh
-var PaddleVersion = "0.3.1"
+var PaddleVersion = "0.4.7"
```

Comment:
why such a jump?

Reply:
Good question! To test it I use `goreleaser` (since Jenkins will automatically download the latest release on Github), so every tested version results in a bump. Not ideal, but works. Ideally we would change it to support pre-releases.